plugins/modules/mount.py

```diff
@@ -734,7 +737,9 @@ def main():
     name = module.params['path']
     changed = False
 
-    if state == 'absent':
+    if state == 'absent_from_fstab':
+        name, changed = unset_mount(module, args)
```

Comment:
I'm not sure if I'm understanding this correctly, but isn't what this PR is adding for `absent_from_fstab` is the same as `absent` (calling `unset_mount` function in https://github.com/ansible-collections/ansible.posix/blob/ecd5ad53e024c1980ea358f543478df2d26df0ce/plugins/modules/mount.py#L324) ?

Reply:
Yes it does the same removal from fstab, but it does not unmount or remove the mountpoint after (which absent does currently).

Reply:
@dkjii-g I realized what the plan is, I've got problem understanding how your code achieves so.